doc/usage/extensions/example_numpy.py

```diff
@@ -266,15 +266,15 @@ def __init__(self, param1, param2, param3):
         self.attr3 = param3  #: Doc comment *inline* with attribute
 
         #: list(str): Doc comment *before* attribute, with type specified
-        self.attr4 = ["attr4"]
+        self.attr4 = ['attr4']
 
         self.attr5 = None
         """str: Docstring *after* attribute, with type specified."""
 
     @property
     def readonly_property(self):
         """str: Properties should be documented in their getter method."""
-        return "readonly_property"
```

Comment:
seems like single quotes aren't even used consistently right now

Reply:
> seems like single quotes aren't even used consistently right now

since it's already inconsistent i would vote for either

1. setting it to the default now
2. using 'preserve' to not touch the quotes at all until a decision is made on a global style

Reply:
but i have to stress- this is a *tiny* nit and i'm in favour of this PR regardless

Reply:
Personally, I had the habit in Python or JS to use single quotes for strings, except docstring, because I don't have to use SHIFT+2 for double quotes. 

However, I understand that some people would say that it's better to use double quotes as it is in C/C++ where single quotes are reserved for ``char``. And since I'm also a C/C++ dev, I cannot really say yes/no for that.

As such, I personally don't mind either of the choices concerning the quotes.